drop-in/src/main/java/com/adyen/checkout/dropin/internal/ui/PaymentMethodListScreen.kt

```diff
@@ -8,53 +8,160 @@
+    PaymentMethodListContent(navigator, viewState)
+}
+
+@OptIn(ExperimentalMaterial3Api::class, ExperimentalMaterial3ExpressiveApi::class)
+@Composable
+private fun PaymentMethodListContent(
+    navigator: DropInNavigator,
+    viewState: PaymentMethodListViewState,
+) {
+    val scrollBehavior = TopAppBarDefaults.exitUntilCollapsedScrollBehavior(rememberTopAppBarState())
     Scaffold(
         containerColor = CheckoutThemeProvider.colors.background,
         topBar = {
-            TopAppBar(
+            MediumFlexibleTopAppBar(
                 colors = TopAppBarDefaults.topAppBarColors(
                     containerColor = CheckoutThemeProvider.colors.background,
+                    scrolledContainerColor = CheckoutThemeProvider.colors.background,
                     navigationIconContentColor = CheckoutThemeProvider.colors.text,
                 ),
-                title = {},
+                title = {
+                    Title(viewState.amount)
+                },
                 navigationIcon = {
-                    val backPressedDispatcher = LocalOnBackPressedDispatcherOwner.current?.onBackPressedDispatcher
                     IconButton(
-                        onClick = { backPressedDispatcher?.onBackPressed() },
+                        onClick = { navigator.back() },
                     ) {
-                        // TODO - String resources
-                        Icon(Icons.Default.Close, "Close")
+                        Icon(Icons.Default.Close, resolveString(CheckoutLocalizationKey.GENERAL_CLOSE))
                     }
                 },
+                scrollBehavior = scrollBehavior,
             )
         },
+        modifier = Modifier.nestedScroll(scrollBehavior.nestedScrollConnection),
     ) { innerPadding ->
         Column(
             modifier = Modifier
                 .padding(innerPadding)
-                .padding(horizontal = Dimensions.Large),
+                .verticalScroll(rememberScrollState()),
         ) {
-            Title("Payment method list")
+            Spacer(Modifier.size(Dimensions.ExtraSmall))
+
+            Body(
+                text = resolveString(CheckoutLocalizationKey.DROP_IN_PAYMENT_METHOD_LIST_DESCRIPTION),
+                color = CheckoutThemeProvider.colors.textSecondary,
+                modifier = Modifier.padding(horizontal = Dimensions.Large),
+            )
+
+            Spacer(Modifier.size(Dimensions.Large))
+
+            viewState.paymentOptionsSection?.let { PaymentOptionsSection(it) }
+        }
+    }
+}
+
+@Composable
+private fun PaymentOptionsSection(
+    paymentOptionsSection: PaymentOptionsSection,
+) {
+    Column {
+        SubHeadlineEmphasized(
+            text = resolveString(paymentOptionsSection.title),
+            modifier = Modifier.padding(horizontal = Dimensions.Large),
+        )
+
+        Spacer(Modifier.size(Dimensions.Small))
+
+        paymentOptionsSection.options.forEach { item ->
```

Comment:
We could extract the list part into another private composable.

Reply:
Since these "options" are part of the `Column` I think it would make sense to keep them here